packages/asc-ui/src/components/FormTitle/FormTitle.stories.mdx

```diff
@@ -22,23 +19,29 @@ export default {
         {storyFn()}
       </div>
     ),
-  ],
+  ]}
+/>
+
+# Form Title
+
+The Form Title component is used for forms, mostly on subdomains of amsterdam.nl. Forms don't have a homepage and can therefore **not** have a `title` in the `<Header>` component.
 
-  parameters: {
-    notes: {
-      markdown: Readme,
-    },
-  },
-}
+[Design System](https://designsystem.amsterdam.nl/7awj1hc9f/p/537bb7-title-component)
 
-export const Default = () => (
-  <ThemeProvider>
+**Component status: stable**
+
+## Default
+
+<Preview>
+  <Story name="Default">
     <FormTitle>This is the title of the FormTitle component</FormTitle>
-  </ThemeProvider>
-)
+  </Story>
+</Preview>
+
+## In context
 
-export const InContext = () => (
-  <ThemeProvider>
```

Comment:
No need for ThemeProvider?

Reply:
Yeah, I am wondering if we need it in these examples. @chrisvanmook is it provided by default in the root somewhere?

Reply:
@jonkoops its provided in `.storybook/config.js` as a decorator, so no need to use that here

Reply:
Great, thanks for the clarification!